mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/AbstractProfileAssociationDao.java

```diff
@@ -69,6 +69,21 @@
     return future;
   }
 
+  @Override
+  public Future<ProfileAssociationCollection> getAll(ContentType masterType, ContentType detailType, String tenantId) {
```

Comment:
Why  these params `ContentType masterType, ContentType detailType` are not used here?
What are they aims?